src/assets/metadata/aws.json

```diff
@@ -1,311 +1,3661 @@
+        },
+        {
+          "name": "password",
+          "displayName": "Password",
+          "description": "Password for the master DB user.",
+          "required": true,
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#password"
+        },
+        {
+          "name": "skip_final_snapshot",
+          "displayName": "Skip final snapshot",
+          "description": "Determines whether a final DB snapshot is created before the DB instance is deleted.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#skip_final_snapshot"
+        },
+        {
+          "name": "publicly_accessible",
+          "displayName": "Publicly accessible",
+          "description": "Controls if instance is publicly accessible.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#publicly_accessible"
+        },
+        {
+          "name": "allow_major_version_upgrade",
+          "displayName": "Allow major version upgrade",
+          "description": "Indicates that major version upgrades are allowed.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#allow_major_version_upgrade"
+        },
+        {
+          "name": "apply_immediately",
+          "displayName": "Apply immediately",
+          "description": "Specifies whether any database modifications are applied immediately, or during the next maintenance window.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#apply_immediately"
+        },
+        {
+          "name": "auto_minor_version_upgrade",
+          "displayName": "Auto minor version upgrade",
+          "description": "Indicates that minor engine upgrades will be applied automatically to the DB instance during the maintenance window.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#auto_minor_version_upgrade"
+        },
+        {
+          "name": "availability_zone",
+          "displayName": "Availability zone",
+          "description": "The Availability Zone for the Relational Database Service instance.",
+          "linkAttribute": "id",
+          "linkRef": "aws_availability_zones",
+          "linkType": "Default",
+          "type": "Link",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#availability_zone"
+        },
+        {
+          "name": "backup_retention_period",
+          "displayName": "Backup retention period",
+          "description": "The days to retain backups for.",
```

Comment:
not sure `for` is needed, I feel like the sentence is not finished
`The days to retain backups` should be enough

Reply:
description from official doc.